docs/docs/reference/configuration-file.mdx

```diff
@@ -3,6 +3,8 @@ title: Configuration File
 sidebar_position: 2
 ---
 
+# Configuration options (Omnibus only)
```

Comment:
(writing) Probably better to avoid parentheses as much as we can. To me it reads like "Oh yeah, almost forgot ..." or "as an aside, ..." and reduces clarity / punchiness. 

```suggestion
# Omnibus configuration options
```

Reply:
Cool, agreed. Always Be Punchy.